core/systems/assignment/gdpr_form.ex

```diff
@@ -55,6 +55,18 @@ defmodule Systems.Assignment.GdprForm do
     }
   end
 
+  @impl true
+  def compose(:confirmation_modal, %{entity: %{consent_agreement: consent_agreement}}) do
+    %{
+      module: Pixel.ConfirmationModal,
```

Comment:
“By skipping the consent, not only will the text be deleted, but the underlying consent agreement and all associated signatures will also be removed. It would be beneficial to clarify this in the body of the confirmation modal, as the current text is static.

Additionally, you may want to implement a check on the consent agreement. If no signatures are present, there is no need to include this clarification in the confirmation modal, as the default text will suffice in such cases.